airflow/providers/amazon/provider.yaml

```diff
@@ -895,6 +896,20 @@ config:
         type: boolean
         example: "True"
         default: "True"
+  aws_auth_manager:
+    description: |
+      This section only applies if you are using the AwsAuthManager. In other words, if you set
+      ``[core] auth_manager = airflow.providers.amazon.aws.auth_manager.aws_auth_manager.AwsAuthManager`` in
+      Airflow's configuration.
+    options:
+      saml_metadata_url:
+        description: |
+          SAML metadata XML file provided by AWS Identity Center.
+          This URL can be found in the AWS Identity Center console. Required.
+        version_added: 10.0.0
+        type: string
+        example: ~
```

Comment:
I think an example of what the url looks like would be helpful

Reply:
Good point